gst/nnstreamer/tensor_query/README.md

```diff
@@ -37,9 +37,9 @@ $ gst-launch-1.0 tensor_query_serversrc ! video/x-raw,width=300,height=300,forma
 ```bash
 $ gst-launch-1.0 v4l2src ! videoconvert ! videoscale !  video/x-raw,width=300,height=300,format=RGB,framerate=30/1 ! tensor_query_client ! videoconvert ! ximagesink
 ```
-#### client 2 (Optional, To test multiple clients)
+#### client 2 (Optional, To test multiple clients, port 0 means any available port number)
```

Comment:
Port 0 is reserved in most systems. To avoid confusion, how about changing it to a negative value?

Reply:
In my opinion, it's okay to use port number 0 to get the port number available in the system. The same method is used in other gstreamer elements. PTAL.

```
$ gst-inspect-1.0 tcpserversrc
...
  port                : The port to listen to (0=random available port)
                        flags: readable, writable
                        Integer. Range: 0 - 65535 Default: 4953 
```